orchestrator/lambdas/layers/heimdall_orgs/heimdall_orgs/org_queue_bitbucket.py

```diff
@@ -74,21 +74,19 @@ def request_orgs(self, start: int = 0) -> Union[dict, None]:
                 url=f"{self.api_url}/projects?limit=100&start={start}", headers=headers, timeout=TIMEOUT
             )
         except requests.exceptions.Timeout:
-            log.error("Request timed out after %ss retrieving orgs for %s", TIMEOUT, self.service)
-            return None
+            raise HeimdallException(f"Request timed out after {TIMEOUT}s retrieving orgs for {self.service}")
         except requests.ConnectionError as e:
-            log.error("Error connecting to %s: %s", self.service, str(e))
-            return None
+            raise HeimdallException(f"Error connecting to {self.service,}: {e}")
```

Comment:
Did the comma get left in?

Reply:
Yes it was left in. let me take it out
How is this still working 🤯